lib/Doctrine/DBAL/Driver/AbstractDriverException.php

```diff
@@ -27,10 +28,15 @@
      * @param string          $message   The driver error message.
      * @param string|null     $sqlState  The SQLSTATE the driver is in at the time the error occurred, if any.
      * @param int|string|null $errorCode The driver specific error code if any.
+     * @param Throwable|null  $previous  The previous exception.
      */
-    public function __construct($message, $sqlState = null, $errorCode = null)
-    {
-        parent::__construct($message);
+    public function __construct(
+        $message,
+        $sqlState = null,
+        $errorCode = null,
+        ?Throwable $previous = null
```

Comment:
Disagree with this change: `AbstractDriverException` is an internal exception type used by DBAL, and shouldn't be used/extended by consumers. This is most likely to be fixed by adding an `@internal`